src/CMakeLists.txt

```diff
@@ -220,6 +220,8 @@ if(HAVE_XIO)
   list(APPEND EXTRALIBS ${XIO_LIBRARY} pthread rt)
 endif(HAVE_XIO)
 
+set(RCU_LIBS ${GPERFTOOLS_TCMALLOC_LIBRARY})
```

Comment:
@dlambrig looks odd--and different from form used in the target_link_libraries(...) locations